hathor/p2p/manager.py

```diff
@@ -132,7 +134,22 @@ def __init__(self, reactor: Reactor, my_peer: PeerId, server_factory: 'HathorSer
     def start(self) -> None:
         self.lc_reconnect.start(5, now=False)
         if settings.ENABLE_PEER_WHITELIST:
-            self.wl_reconnect.start(30)
+            self.start_whitelist_reconnect()
+
+    def start_whitelist_reconnect(self) -> None:
```

Comment:
I'd rename it to `start_whitelist_update(...)`. It seems simpler to name it after what it does. The timeout is just part of the starting process.

Reply:
I think I prefer `start_whitelist_reconnect` because the name of the `LoopingCall` variable is `wl_reconnect` and this method starts this looping call.

Reply:
Makes sense.